cerner/terra-framework
# Modal manager redwood styling

### Summary
This PR adds redwood styling to the modal manager and adds wdio screenshots for that component.

**NOTE: the redwood css styles that changed are border radius, background color and box-shadow css props of the disclosure modal, the content of the disclosure component is not impacted by the change. The wdio screenshots will need to be updated once the content components become redwood-styled.**

### Testing

This change was tested using:

- [x] WDIO
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] No tests are needed

Steps to test:
1. Run `npm ci` to build and generate css styles for all themes.
2. Switch to redwood theme.
3. In developer tool observe that redwood values were applied to background-color, box-shadow, border-radius, and color props to the modal manager `<div>`.

<img width="1563" alt="Screenshot 2024-01-18 at 3 42 30 PM" src="https://github.com/cerner/terra-framework/assets/119358186/3ef46341-7405-4931-a47f-6e0a028838a7">

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Functional review

**This PR resolves:**
UXPLATFORM-10006
